interfaces/utils/path_patterns.go

```diff
@@ -137,6 +137,9 @@ func createRegex(pattern string, glob GlobFlags) (string, error) {
 			if currentGroupLevel > 0 {
 				itemCountInGroup[currentGroupLevel]++
 				regex += "|"
+			} else if allowCommas {
+				// treat commas outside of groups as literal commas
+				regex += regexp.QuoteMeta(",")
```

Comment:
Is `QuoteMeta` required here?

Reply:
It's not. I initially just had `regex += ","` but decided to add it just to be safe, as alexmurray had a comment about commas allowing breaking out of rules by being treated as a regex rule separator. This isn't an actual risk, as the whole regex is surrounded by quotes. I can revert this back to `regex += ","`. Thanks!